### DISCUSSION TITLE
Enabling Events on TextArea Object

### DISCUSSION THREAD

**Discussion by jcoppage** [rexrainbow/phaser3-rex-notes#204] [Q&A]
Hi there!  Love the plugins, I can't get enough of them.  
Question: I am trying to add an .on('pointerover') and .on('pointerout') event onto my textArea object, which works when I set textArea.setInteractive(), but the problem is I am no longer able to use the scroller and slider features of the object unless I set it to disableInteractive().  But I always want it to be setInteractive() so it can detect if the pointer is over the object.  Is there anyway around this? The only way I can think of is just checking the pointer position in the update method.  

Thanks for your hard work.

Jeff

**Comment by rexrainbow** [comment]
The scroller area and text game object are overlapped, you might try to disable [topOnly](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/touchevents/#top-only) to solve it.

**Comment by jcoppage** [comment]
That works perfectly.  Thank you Rex.
